sdk/core/azure-core-opentelemetry/NOTICE.txt

```diff
@@ -0,0 +1,135 @@
+azure-core
+
+NOTICES AND INFORMATION
+Do Not Translate or Localize
```

Comment:
why? probably some legal thing .. still .. sad

Reply:
Corporate Governance thingy. Controlled by the cgmanifest.json file.